playwright/frame.py

```diff
@@ -131,6 +131,14 @@ async def querySelector(self, selector: str) -> Optional[ElementHandle]:
             await self._channel.send("querySelector", dict(selector=selector))
         )
 
+    async def querySelectorAll(self, selector: str) -> List[ElementHandle]:
+        return list(
+            map(
+                cast(ElementHandle, from_nullable_channel),
```

Comment:
Should be from_channel, there can be no nulls.

Reply:
thx for fixing